app/controllers/declaration/DestinationCountriesController.scala

```diff
@@ -178,20 +180,20 @@ class DestinationCountriesController @Inject()(
   private def removeRoutingCountry(keys: Seq[String], cachedData: DestinationCountriesStandard)(
     implicit request: JourneyRequest[_]
   ): Future[Result] = {
-    val key = keys.headOption.getOrElse("")
+    val key = keys.headOption.getOrElse("").toInt
```

Comment:
`getOrElse` will return you String. Empty string cannot be cast to Int
```
scala> "".toInt
java.lang.NumberFormatException: For input string: ""
  at java.lang.NumberFormatException.forInputString(NumberFormatException.java:65)
  at java.lang.Integer.parseInt(Integer.java:592)
  at java.lang.Integer.parseInt(Integer.java:615)
  at scala.collection.immutable.StringLike.toInt(StringLike.scala:304)
  at scala.collection.immutable.StringLike.toInt$(StringLike.scala:304)
  at scala.collection.immutable.StringOps.toInt(StringOps.scala:33)
  ... 28 elided
```

Reply:
Thank you for your piece od advice. I've changed it to throw an exception explicitly. It is more clear now, that this is expected behaviour.